build/yaml/templates/versioning-steps.yml

```diff
@@ -0,0 +1,15 @@
+steps:
+  - powershell: |
+      "Deployment Ring = $(DeploymentRing)";
```

Comment:
Is this first line just for logging purposes?

Reply:
Yes, helps to validate when the builds are running like so:
![image](https://user-images.githubusercontent.com/43043272/108554729-5e2d5200-72a9-11eb-857e-8bf606a7dcc7.png)

Reply:
Excellent!